lib/pure/mersenne.nim

```diff
@@ -7,12 +7,26 @@
 #    distribution, for details about the copyright.
 #
 
+## The Mersenne Twister random number generator.
```

Comment:
if possible (in this or future PR), add a comment explaining when you'd prefer std/random vs std/mersenne (quality/statistical properties of random number generator, period of the prng, and performance comparison)

Reply:
I honestly have no idea, I'd need to do further research for that.